### PR TITLE
@craigspaeth => Support reference shows on the artist page

### PR DESCRIPTION
**Pull request by mzikherman**
Now that the bulk of the wiring up work was done in Gravity/Galaxy/Metaphysics, exposing ref shows on the artist page is a breeze.

### CODE REVIEW

File: apps/artist/queries/cv.coffee [hunk 1]
```diff
@@ -3,10 +3,10 @@ module.exports =
   query artist($artist_id: String!, $shows: Boolean!, $articles: Boolean!) {
     artist(id: $artist_id) {
       ... on Artist @include(if: $shows) {
-        group_shows: shows(at_a_fair: false, solo_show:false, sort: start_at_desc, size: 99) {
```
Comment (mzikherman) [inline apps/artist/queries/cv.coffee]: Include reference shows (in addition to regular shows).

File: apps/artist/queries/show_fragment.coffee [hunk 2]
```diff
@@ -9,9 +9,7 @@ module.exports =
         name
       }
     }
-    location {
-      city
-    }
+    city
```
Comment (mzikherman) [inline apps/artist/queries/show_fragment.coffee]: Support the new way to get the `city`, either from Galaxy or Artsy. The [display helper](https://github.com/artsy/force/blob/93fc7773a6d509955bc1e476b5fa28a8f8ee80cf/components/show_cell/helpers.coffee#L14-L17) for the show cell is already updated to support all the styles of querying for locations.

**Comment by mzikherman** [comment]
Looks like:

screen shot 2016-10-10 at 12 54 04 pm

**Comment by mzikherman** [comment]
Closes https://github.com/artsy/force/issues/224

File: lib/global_client_setup.coffee [hunk 3]
```diff
@@ -96,6 +96,8 @@ disableRightClick = ->
   $(document).on 'contextmenu', (e) ->
     # Grenade for selecting any images with the name 'artwork' in their,
     # or their parent's, class
-    disable = $(e.target).is('[class*=artwork]') or
-              $(e.target).parent().is('[class*=artwork]')
+    $target = $(e.target)
+    return unless $target.is('img')
```
Comment (craigspaeth) [inline lib/global_client_setup.coffee]: Thanks for that!

**Comment by craigspaeth** [comment]
Very nice! Love how much GraphQL cleans up the data logic!

**Pull request merged**